### DISCUSSION TITLE
Load group of images into gallery not related to initial thumbnails

### DISCUSSION THREAD

**Discussion by swstyles** [sachinchoolur/lightGallery#1071] [General]
Hello,

Most of the examples I have found load images related to the initial thumbnails shown on the page. I would like to load a set of images related only to the thumbnail clicked. For example I have 3 cats, one brown, black and white. When the user clicks on the white cat, I don't want to see a gallery of the 3 cats shown initially. I would like to load let's say 6 images related to the white cat.

Hope that made sense.

TIA

**Comment by sachinchoolur** [comment]
Hey,
You can use dynamic mode to open gallery with a set of images.
Instead of the button on the dynamic mode demo to launch gallery, you can
display the thumbnail itself. I’ll provide a demo by Tomorrow morning if
required.

On Tue, 29 Jun 2021 at 10:45 PM, Daniel Magallanes ***@***.***>
wrote:

> Hello,
>
> Most of the examples I have found load images related to the initial
> thumbnails shown on the page. I would like to load a set of images related
> only to the thumbnail clicked. For example I have 3 cats, one brown, black
> and white. When the user clicks on the white cat, I don't want to see a
> gallery of the 3 cats shown initially. I would like to load let's say 6
> images related to the white cat.
>
> Hope that made sense.
>
> TIA
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/sachinchoolur/lightGallery/discussions/1071>, or
> unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ABSICUSNIQYJT2Z4SX3R4OLTVH5UJANCNFSM47QVHAWA>
> .
>
-- 
*Regards,*
*SACHIN N*


*[image: Github] <https://github.com/sachinchoolur>  [image:
@sachinchoolur] <https://twitter.com/sachinchoolur>  [image: linkedin]
<https://www.linkedin.com/in/sachinchoolur>*

**Reply by swstyles** [reply]
If you could provide a demo, I would really appreciate it.

Thank you for all the work and effort you have put into Lightgallery.

**Reply by sachinchoolur** [reply]
Demo - https://codepen.io/sachinchoolur/pen/vYmEyRE

Basically, you need to create 3 galleries and clicking on each thumbnail, open the respective gallery 

Note: currently, the zoom from origin option is not available for dynamic mode. But, I'll add the support soon

**Reply by swstyles** [reply]
Thank you for your help on this.

**Reply by swstyles** [reply]
I ended up buying 2 licenses of this. Hopefully you continue to build and grow this terrific tool.

**Reply by sachinchoolur** [reply]
Thank you so much
